.cloudbuild/prombench.yaml

```diff
@@ -0,0 +1,487 @@
+steps:
+  - id: "Verifying GitHub comment and user permissions"
+    name: bash
+    env:
+    - '_COMMENTER=$_COMMENTER'
```

Comment:
Is `_` prefix is some pattern? It's kind of unreadable and odd.  If there's reason, do you mind adding comment why?